vhdbuilder/packer/install-dependencies.sh

```diff
@@ -70,7 +70,7 @@ if [[ ${CONTAINER_RUNTIME:-""} == "containerd" ]]; then
   downloadTeleportdPlugin ${TELEPORTD_PLUGIN_DOWNLOAD_URL} "0.5.0"
 else
   CONTAINER_RUNTIME="docker"
-  MOBY_VERSION="19.03.12"
+  MOBY_VERSION="19.03.14"
```

Comment:
required so the var is defined below, but we can definitely clean up the variable handling here.